proxmox/virtual_environment_vm_types.go

```diff
@@ -229,6 +229,7 @@ type VirtualEnvironmentVMCreateRequestBody struct {
 	ACPI                 *CustomBool                  `json:"acpi,omitempty" url:"acpi,omitempty,int"`
 	Agent                *CustomAgent                 `json:"agent,omitempty" url:"agent,omitempty"`
 	AllowReboot          *CustomBool                  `json:"reboot,omitempty" url:"reboot,omitempty,int"`
+	Args                 *string                      `json:"args,omitempty" url:"args,omitempty"`
```

Comment:
See the linter error below about `KVMArguments` -- that's actually a good catch, i wouldn't notice that 😅
I think we can totally reuse this field instead of the new one, but I leave it to you which one to keep.